src/transformers/models/cvt/configuration_cvt.py

```diff
@@ -0,0 +1,145 @@
+# coding=utf-8
+# Copyright Anugunj Naman and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" CvT model configuration """
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+CVT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "microsoft/cvt-21-384x384": "https://huggingface.co/microsoft/cvt-21-384x384-IN-1k/resolve/main/config.json",
+    # See all CvT models at https://huggingface.co/models?filter=cvt
+}
+
+
+class CvTConfig(PretrainedConfig):
```

Comment:
I would rename `CvTConfig` to `CvtConfig` (and everywhere else).
```suggestion
class CvtConfig(PretrainedConfig):
```